fastcrypto-tbls/src/dkg.rs

```diff
@@ -176,7 +182,15 @@ where
 
     /// 4. Create the first message to be broadcasted.
     pub fn create_message<R: AllowedRng>(&self, rng: &mut R) -> Message<G, EG> {
-        debug!("Creating message for party {}", self.id);
+        let vss_pk = self.vss_sk.commit();
```

Comment:
not sure how expensive these calls are but consider using 

`if tracing::enabled!(tracing::Level::INFO)` or equivalent if you want to avoid the work unless the logs are turned on

Reply:
We need that value, I moved it from the end of the function